rbac/management/role/definer.py

```diff
@@ -84,3 +84,36 @@ def seed_roles(tenant, update=False):
             role_list = data.get("roles")
             _update_or_create_roles(tenant, role_list, update)
     return tenant
+
+
+def seed_permissions(tenant):
+    """For a tenant update or create defined permissions."""
+    permission_directory = os.path.join(settings.BASE_DIR, "management", "role", "permissions")
+    permission_files = [
+        f
+        for f in os.listdir(permission_directory)
+        if os.path.isfile(os.path.join(permission_directory, f)) and f.endswith(".json")
+    ]
+    # current_permission_ids = set()
+
+    with tenant_context(tenant):
+        for permission_file_name in permission_files:
+            permission_file_path = os.path.join(permission_directory, permission_file_name)
+            app_name = os.path.splitext(permission_file_name)[0]
+            with open(permission_file_path) as json_file:
+                data = json.load(json_file)
+                for resource, operations in data.items():
+                    for operation in operations:
+                        permission, created = Permission.objects.get_or_create(
```

Comment:
I think we may be want to do something here like an `update_or_create` [1] to ensure we're getting all fields set? It seems this migration [2] is only setting the `permission` field on `Permission`, so we end up with `Permission` records without the other fields populated. 

[1] https://docs.djangoproject.com/en/dev/ref/models/querysets/#update-or-create
[2] https://github.com/RedHatInsights/insights-rbac/pull/291/files#diff-abf50ea78e7ec39af811f223d8c6aa53R15